pymc/logprob/binary.py

```diff
@@ -136,3 +136,55 @@ def comparison_logprob(op, values, base_rv, operand, **kwargs):
         logcdf.name = f"{base_rv_op}_logcdf"
 
     return logprob
+
+
+class MeasurableBitwise(MeasurableElemwise):
+    """A placeholder used to specify a log-likelihood for a bitwise operation RV sub-graph."""
+
+    valid_scalar_types = (Invert,)
+
+
+@node_rewriter(tracks=[invert])
+def find_measurable_bitwise(fgraph: FunctionGraph, node: Node) -> Optional[List[MeasurableBitwise]]:
+    rv_map_feature = getattr(fgraph, "preserve_rv_mappings", None)
+    if rv_map_feature is None:
+        return None  # pragma: no cover
+
+    if isinstance(node.op, MeasurableBitwise):
+        return None  # pragma: no cover
+
+    base_var = node.inputs[0]
+    if not (
+        base_var.owner
+        and isinstance(base_var.owner.op, MeasurableVariable)
+        and base_var not in rv_map_feature.rv_values
+        and base_var.dtype.startswith("bool")
```

Comment:
Nitpick: What about separating this check? 

The other checks are basically done by every rewrite and this one is specific to the invert. When parsing what this rewrite is doing it would help to see it in a separate if block.

Reply:
Definitely, that makes more sense.